source/Jobbr.Server.WebAPI/Infrastructure/WebHost.cs

```diff
@@ -12,84 +18,128 @@ namespace Jobbr.Server.WebAPI.Infrastructure
     /// </summary>
     public class WebHost : IJobbrComponent
     {
-        /// <summary>
-        /// The logger.
-        /// </summary>
-        private static readonly ILog Logger = LogProvider.For<WebHost>();
-
-        /// <summary>
-        /// The dependency resolver.
-        /// </summary>
-        private readonly IJobbrServiceProvider dependencyResolver;
-
-        /// <summary>
-        /// The configuration for this component
-        /// </summary>
-        private readonly JobbrWebApiConfiguration configuration;
-
-        /// <summary>
-        /// The webhost that serves for the OWIN WebAPI component
-        /// </summary>
-        private IDisposable webHost;
+        private WebApplication _webApp;
+        private readonly ILogger _logger;
+        private readonly ILoggerFactory _loggerFactory;
```

Comment:
I don't think the factory needs to be stored

Reply:
it is injected in JsonTypeConverter